<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetClass</class>
 <widget class="QWidget" name="QtWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/person2.png</normaloff>:/image/person2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="toolButton_2">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/image/password1.png</normaloff>:/image/password1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>账号:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_passw"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_passward">
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_accounts"/>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushBut_logOn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBut_logIn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
